Added configurable functionality

frank-zimmermann opened the pull request:
Created 3 parameters in the project file, now it is possible to:
- toggle security header and footer between **yes** and **no**
- set the security text
- set the header and footer background color

It is then also possible to set the values in the transformation scenario.

Fixes https://github.com/DeepBlueCLtd/Fi3ldMan/issues/89

frank-zimmermann committed:
Added confgirable functionality

IanMayo commented:
Nice work @frank-zimmermann.

The next step is to include the headers and footers on the other pages.

A reminder. This is my preferred order of supplementing the template:
![image](https://github.com/DeepBlueCLtd/LegacyMan/assets/1108513/6c9433f4-b235-4505-9f5c-0a8f6dfdc369)

But, if XSL is the only option I'm happy to go with that.

IanMayo committed:
change name of parameter

frank-zimmermann committed:
Header and footer for topic pages

- separate extension into option file
- own xsl for footer and header to import them

IanMayo reviewed (commented):
Hello @frank-zimmermann , since the two custom xsl files are almost identical (only whitespace differences), is it an option to just use the one file in both extensions?

frank-zimmermann commented:
> Hello @frank-zimmermann , since the two custom xsl files are almost identical (only whitespace differences), is it an option to just use the one file in both extensions?

No, we need to have this in both files. 
![image](https://github.com/DeepBlueCLtd/LegacyMan/assets/49056485/f39f29a6-2abc-4be4-b60d-830e8af73ee7)
Both extension points, `xslt/customMainPage.xsl` and `xslt/customTopicPage.xsl` are needed. But both use the same header and footer information. So, we are very flexible in adapting the code by using won footer and header templates.
